homework1/Task1.java

```diff
@@ -2,8 +2,17 @@
 
 public class Task1 {
     public static void main(String[] args) {
+        System.out.println("****  ****      *      *   *");
+        System.out.println("*     *  *     * *     ** **");
+        System.out.println("**    ****    *   *    * * *");
+        System.out.println("*     *      *******   *   *");
+        System.out.println("****  *     *       *  *   *");
+        System.out.println("\n");
```

Comment:
It's not necessary to use the "\n" for a new line.
When you use `System.out.println()` you put \n at the end of the line.